.github/workflows/create-release-official.yml

```diff
@@ -68,11 +66,9 @@ jobs:
       - name: Protect image
         run: |
           container_id=${{env.container_id}}
-          docker exec -e DOCKER_PUSH_TARGET -e DOCKER_REGISTRY -e AZURE_TENANT_ID -e AZURE_CLIENT_ID -e AZURE_CLIENT_SECRET -e AZURE_SUBSCRIPTION_ID "$container_id" task controller:acr-protect-image
+          docker exec -e GITHUB_ACTIONS -e DOCKER_PUSH_TARGET -e DOCKER_REGISTRY -e AZURE_TENANT_ID -e AZURE_SUBSCRIPTION_ID "$container_id" task controller:acr-protect-image
```

Comment:
Where's this `GITHUB_ACTIONS` var is sourced from?

Reply:
GitHub itself includes it in all the action contexts. It's how you can easily tell if you're running in the context of an action versus locally.